trunk/SUAVE/Components/Wings/Control_Surface.py

```diff
@@ -20,6 +20,8 @@ def __defaults__(self):
         self.tag                   = 'control_surface'
         self.span                  = 0.0
         self.span_fraction         = 0.0
+        self.chord_fraction        = 0  # Added by Matthew 
+        self.gain                  = 0  # Added by Matthew
```

Comment:
These are variables that AVL require. They are essentially how much of the airfoil chord on the wing is the control surface and the orientation. A gain of 1 is a downward angled flap. A gain od -1 is a upward angled flap.